modules/enterprise/server/jar/src/main/java/org/rhq/enterprise/server/rest/SetCallerInterceptor.java

```diff
@@ -72,6 +81,14 @@ public Object setCaller(InvocationContext ctx) throws Exception {
 
         Subject caller=null;
         java.security.Principal p = ejbContext.getCallerPrincipal();
+        if (!startupBean.isInitialized()) {
+            String notInitMessage = "Tried to call REST endpoint but the server has not finished to startup";
+            log.warn(notInitMessage);
```

Comment:
Do we even want to log this? If we feel this message may be useful, perhaps it should only get logged at DEBUG level? Just seems that "expected" failures such as "I'm not running yet" don't warrant a log message in the log of the thing that "is not running yet."

Log messages should be considered valid in exceptional or informational state change events. "I am starting..." "I have started." "I am not going to start due to failure..." etc.

If there is no way to tell that the server had not started from the context of the surrounding log messages then perhaps a DEBUG message would be helpful in those rare cases that someone cannot figure out that the REST service is unavailable because the server is not running.